### PR TITLE
Remove 'verify.fileAfterCodeFix', use 'verify.codeFix'

### PR DESCRIPTION
**Pull request by andy-ms**
There's no reason to have two separate methods for testing the same thing, and `verify.codeFix` strictly tests more, since it tests the description and whitespace.

### CODE REVIEW

File: tests/cases/fourslash/codeFixInferFromUsagePropertyAccessJS.ts [hunk 1]
```diff
@@ -16,7 +16,10 @@
 ////    return x.y.z
 ////}
 
-verify.fileAfterCodeFix(
+verify.codeFix({
+    description: "Infer parameter types from usage",
+    index: 2,
```
[the remark this answers is not in the document]
Comment (andy-ms) [reply tests/cases/fourslash/codeFixInferFromUsagePropertyAccessJS.ts]: `verify.codeFix` does the same thing -- using the same (duplicated) code.
